lib/ansible/modules/windows/win_rds_cap.py

```diff
@@ -0,0 +1,122 @@
+  - Kevin Subileau (@ksubileau)
+options:
+  name:
+    description:
+      - Name of the connection authorization policy.
+    required: yes
+  state:
+    description:
+      - The state of connection authorization policy.
+      - If C(absent) will ensure the policy is removed.
+      - If C(present) will ensure the policy is configured and exists.
+      - If C(enabled) will ensure the policy is configured, exists and enabled.
+      - If C(disabled) will ensure the policy is configured, exists, but disabled.
+    choices: [ absent, present, enabled, disabled ]
+    default: present
+  auth_method:
+    description:
+      - Specifies how the RD Gateway server authenticates users.
+      - When a new CAP is created, the default value is C(password).
+    choices: [ password, smartcard, both, none ]
+  order:
+    description:
+      - Evaluation order of the policy.
+      - The CAP in which I(order) is set to a value of '1' is evaluated first.
+      - By default, a newly created CAP will take the first position.
+      - If the given value exceed the total number of existing policies,
+        the policy will take the last position but the evaluation order
+        will be capped to this number.
+    type: int
+  session_timeout:
+    description:
+      - The maximum time, in minutes, that a session can be idle.
+      - A value of zero disables session timeout.
+    type: int
+  session_timeout_action:
+    description:
+      - The action the server takes when a session times out.
+      - 'C(disconnect): disconnect the session.'
+      - 'C(reauth): silently reauthenticate and reauthorize the session.'
+    choices: [ disconnect, reauth ]
+    default: disconnect
+  idle_timeout:
+    description:
+      - Specifies the time interval, in minutes, after which an idle session is disconnected.
+      - A value of zero disables idle timeout.
+    type: int
+  allow_only_sdrts_servers:
+    description:
+      - Specifies whether connections are allowed only to Remote Desktop Session Host servers that
+        enforce Remote Desktop Gateway redirection policy.
+    type: bool
+  user_groups:
+    description:
+      - A list of user groups that is allowed to connect to the Remote Gateway server.
+      - Required when a new CAP is created.
+    type: list
+  computer_groups:
+    description:
+      - A list of computer groups that is allowed to connect to the Remote Gateway server.
+    type: list
+  redirect_clipboard:
+    description:
+      - Allow clipboard redirection.
+    type: bool
+  redirect_drives:
+    description:
+      - Allow disk drive redirection.
+    type: bool
+  redirect_printers:
+    description:
+      - Allow printers redirection.
+    type: bool
+  redirect_serial:
+    description:
+      - Allow serial port redirection.
+    type: bool
+  redirect_pnp:
+    description:
+      - Allow Plug and Play devices redirection.
+    type: bool
+requirements:
+  - Windows Server 2008R2 (6.1) or higher with RDS features
```

Comment:
Would be good to specify the name of the feature as shown in `Get-WindowsFeature`